gemfiles/rack_1.5.5.gemfile

```diff
@@ -2,7 +2,7 @@
 
 source 'https://rubygems.org'
 
-gem 'rack', '1.5.2'
+gem 'rack', '1.5.5'
```

Comment:
I'm going to remove this "appraisal", as it was added specifically for fixing #1066(missing patch version in `rack` 1.5.2).
any objections? 

moreover, now I think it does not worth it :)